SinaiStCatherine/SinaiEt004.xml

```diff
@@ -227,6 +227,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                       <locus from="15ra" to="21vb"/>
                         <title type="incomplete" ref="LIT2362Songof"/>
                       <textLang mainLang="gez"/>
+                              <note>Followed by two other short texts: Weddase be ehud and </note>
```

Comment:
this note looks incomplete :)

Reply:
not changed yet? and repeated below under 1.1.2?

Reply:
not changed